resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -14,10 +14,10 @@ These instructions will walk you through installing the required Data Science so
 - uses English as the default language
```

Comment:
This is because we have had problems with older laptops, and students push the limits of this. I honestly think laptops made in early 2010's will face two problems:
- not enough RAM and hard disk space
- might not be able to update to the latest OS we require

Reply:
Can you add this to next weeks academic meeting agenda? This is a bigger issue that I do not feel comfortable for just you and I making the decision to change it.